src/main/java/org/reaktivity/nukleus/tls/internal/stream/ClientStreamFactory.java

```diff
@@ -317,6 +322,7 @@ private RouteFW wrapRoute(
         private long applicationTraceId;
 
         private ClientAcceptStream(
+            SSLEngine tlsEngine,
```

Comment:
Source#handleBegin() creates ClientAcceptStream and hands over BEGIN to newly created stream. Since BEGIN is following immediately, is there any benefit in delaying the creation of SSLEngine ?

Reply:
Not creating Supplier<SSLEngine> as it was already a field, now becomes final field